src/lib.rs

```diff
@@ -850,7 +855,7 @@ line1
         );
 
         // Check `expect![[  ]]` as well.
-        assert_eq!(locate_end("]]"), Some(0));
+        assert_eq!(locate_end("[]]"), Some(2));
```

Comment:
The tests should have `[` at the beginning.

Reply:
- Is the original test still relevant?
- _If so,_ should we leave in both assertions?
- _If so,_ what should it equal?

```suggestion
        assert_eq!(locate_end("]]"), todo!());
        assert_eq!(locate_end("[]]"), Some(2));
```

Reply:
updated

Reply:
Wait, I think actually it's not relevant. Because `locate` is used for `update`. But when `update` is called, the syntax has to be correct.